tutorials/eth-api/hardhat-start-to-end.md

```diff
@@ -494,89 +480,81 @@ While it's possible to verify smart contracts on the [Moonscan website](https://
 
 Before beginning the contract verification process, you'll need to [acquire a Moonscan API Key](/builders/build/eth-api/verify-contracts/etherscan-plugins/#generating-a-moonscan-api-key){target=\_blank}. Note that Moonbeam and Moonbase Alpha use the same [Moonbeam Moonscan](https://moonscan.io/){target=\_blank} API key, whereas you'll need a distinct API key for [Moonriver](https://moonriver.moonscan.io/){target=\_blank}.
 
-To verify the contract, you will run the `verify` command and pass in the network where the `DelegationDao` contract is deployed, the address of the contract, and the two constructor arguments that you specified in your `deploy.js` file, namely, the address of the target collator and the address you deployed the smart contract with (sourced from the `hardhat.config.js` file).
+To verify the contract, you will run the `ignition verify` command and pass the deployment-id, the `YOUR-NAME-HERE` parameter in the prior step.
```

Comment:
`deployment-id`

Reply:
ah ok so we aren't actually using the flag, maybe you should say pass the value you provided to the `deployment-id` flag in the previous section/step